src/examples/IK/exampleIK.cpp

```diff
@@ -273,177 +262,137 @@ int main()
-                                                                      // the number of degrees of
-                                                                      // freedom
-    initialJointPositions.setZero(); // Set initial joint positions to zero
-    gravity << 0.0, 0.0, -9.81; // Assign values to the gravity vector
-    basePose.setIdentity(); // Set the base pose to the identity matrix
-    baseVelocity.setZero(); // Set base velocity to zero
 
-    auto paramHandler
-        = std::make_shared<BipedalLocomotion::ParametersHandler::YarpImplementation>(); // Create a
-                                                                                        // parameter
-                                                                                        // handler
+    // Resize the joint variables based on the number of degrees of freedom
 
-    // Set parameters from a configuration file
-    if (!paramHandler->setFromFile(getConfigPath() + "/exampleIK.ini")) // Check if setting
-                                                                        // parameters from file is
-                                                                        // successful
-    {
-        BiomechanicalAnalysis::log()->error(
-            "Cannot configure the parameter handler"); // Log error
-                                                       // if
-                                                       // configuration
-                                                       // fails
-        return 1; // Return error code
-    }
+    jointPos.resize(kinDyn->getNrOfDegreesOfFreedom()); // Resize the joint position vector based on the number of degrees of freedom
+    jointPos2.resize(kinDyn->getNrOfDegreesOfFreedom()); // Resize another joint position vector based on the number of degrees of freedom
+    jointVelocities.resize(kinDyn->getNrOfDegreesOfFreedom()); // Resize the joint velocity vector based on the number of deg of freedom
+    jointPositions.resize(kinDyn->getNrOfDegreesOfFreedom()); // Resize the joint position vector based on the number of deg of freedom
+    initialJointPositions.resize(kinDyn->getNrOfDegreesOfFreedom()); // Resize the initial joint pos vec based on the n. of deg of freedom
+    initialJointVelocities.resize(kinDyn->getNrOfDegreesOfFreedom()); // Resize the initial joint vel vec based on the n. of deg of freedom
+
+    //*************************************************************************************************************************************
+
+    //*************************************************************************************************************************************
+    // Load kinematic data from Matlab file human_data
+    //*************************************************************************************************************************************
+
+    matioCpp::File file2("/path/to/human_data1.mat"); // Create another file object to read data from
```

Comment:
maybe here we can read the path to the matlab file from the [ini file](https://github.com/ami-iit/biomechanical-analysis-framework/blob/debug-tpose/src/examples/IK/exampleIK.ini).

Reply:
Done with commit https://github.com/ami-iit/biomechanical-analysis-framework/pull/31/commits/575da1ffecd6145855cad2d5019c3e90325b45b3